implementation.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var ES = require('es-abstract/es6');
+
+/*! https://mths.be/array-from v0.2.0 by @mathias */
+module.exports = function from(arrayLike) {
+	var defineProperty = (function() {
```

Comment:
aha, that's true. we could use it to detect support, at least, with the `supportsDescriptors` property